pkg/app/web/src/modules/deployments/index.test.ts

```diff
@@ -1,3 +1,5 @@
+global.Date.now = jest.fn(() => 0);
```

Comment:
I just want to mock `Date.now` by setting this before constructing modules to be imported.
Tell me if there is anything better.

Reply:
Don't have idea for other way to archive that, so the current way is lgtm 👍